pyproject.toml

```diff
@@ -37,6 +37,7 @@ dev = [
     "black",
     "build",
     "bumpver",
+    "django-webpack-loader",
```

Comment:
I think the webpack loader is in fact a prod dependency, since it's required to parse the webpack stats and derive the file names of the bundled files from it...